test/unit/test_grid_extrusion.py

```diff
@@ -604,56 +604,56 @@ def test_single_fracture(self):
         mg_new = d["mortar_grid"]
 
         # Check that the old and new grid have
-        bound_faces_old = mg.master_to_mortar_int().tocoo().col
+        bound_faces_old = mg.primary_to_mortar_int().tocoo().col
         # We know from the construction of the grid extruded to 3d that the faces
         # on the fracture boundary will be those in the 2d grid stacked on top of each
         # other
         known_bound_faces_new = np.hstack(
             (bound_faces_old, bound_faces_old + g_2.num_faces)
         )
-        bound_faces_new = mg_new.master_to_mortar_int().tocoo().col
+        bound_faces_new = mg_new.primary_to_mortar_int().tocoo().col
         self.assertTrue(
             np.allclose(np.sort(known_bound_faces_new), np.sort(bound_faces_new))
         )
 
-        low_cells_old = mg.slave_to_mortar_int().tocoo().col
+        low_cells_old = mg.secondary_to_mortar_int().tocoo().col
         known_cells_new = np.hstack((low_cells_old, low_cells_old + g_1.num_cells))
 
-        cells_new = mg_new.slave_to_mortar_int().tocoo().col
+        cells_new = mg_new.secondary_to_mortar_int().tocoo().col
 
         self.assertTrue(np.allclose(np.sort(known_cells_new), np.sort(cells_new)))
 
-        # All mortar cells should be associated with a face in master
-        mortar_cells_in_range_from_master = np.unique(
-            mg_new.master_to_mortar_int().tocoo().row
+        # All mortar cells should be associated with a face in primary
+        mortar_cells_in_range_from_primary = np.unique(
+            mg_new.primary_to_mortar_int().tocoo().row
         )
-        self.assertTrue(mortar_cells_in_range_from_master.size == mg_new.num_cells)
+        self.assertTrue(mortar_cells_in_range_from_primary.size == mg_new.num_cells)
         self.assertTrue(
-            np.allclose(mortar_cells_in_range_from_master, np.arange(mg_new.num_cells))
+            np.allclose(mortar_cells_in_range_from_primary, np.arange(mg_new.num_cells))
         )
 
-        # All mortar cells should be in the range from slave
-        mortar_cells_in_range_from_slave = np.unique(
-            mg_new.slave_to_mortar_int().tocoo().row
+        # All mortar cells should be in the range from secondary
+        mortar_cells_in_range_from_secondary = np.unique(
+            mg_new.secondary_to_mortar_int().tocoo().row
         )
-        self.assertTrue(mortar_cells_in_range_from_slave.size == mg_new.num_cells)
+        self.assertTrue(mortar_cells_in_range_from_secondary.size == mg_new.num_cells)
         self.assertTrue(
             np.allclose(
-                np.sort(mortar_cells_in_range_from_slave), np.arange(mg_new.num_cells)
+                np.sort(mortar_cells_in_range_from_secondary), np.arange(mg_new.num_cells)
             )
         )
 
         g_1_new = gb_new.grids_of_dimension(2)[0]
```

Comment:
g_secondary_new?

Reply:
That's a 1 I think. I'm not sure if this should be `g_primary_new` of `g_secondary_new` to be honest.

Reply:
I will leave this as it is since I'm not sure. Either way, I guess this has to do with #478